components/Downloads/ChangelogModal/index.module.css

```diff
@@ -0,0 +1,71 @@
+/* What color should the svg be? */
```

Comment:
Wdym? Which SVG are we talking about?

Reply:
The arrow attached to the link sorry

Reply:
I think you can check what components such as NavItem do, but in general the colour of the SVG should be on Figma

Reply:
Wow I thought that "Start Contributing" link was green on Figma haha I might need a colorblind test